docs/installation_quickstart.rst

```diff
@@ -33,28 +40,24 @@ For more advanced usage of ``ddtrace-run`` refer to the documentation :ref:`here
 
 To find out how to trace your own code manually refer to the documentation :ref:`here<basic usage>`.
 
+Profiling
+~~~~~~~~~
+
+Getting started for profiling is as easy as prefixing your python entry-point
+command with ``pyddprofile``.
```

Comment:
aside: we should probably make `pyddtrace` or something too, keep `ddtrace-run` for backwards compatibility, but figure out the pattern we want moving forward.

Reply:
Agreed. `pyddprofile` is a wrapper for `python` itself while `ddtrace-run` can run any program.

We may want both actually. You can't really run e.g. `gunicorn` with `pyddprofile`.

Reply:
got it, but I also don't _love_ that we have `ddtrace-run` which doesn't indicate that it is Python specific, which it is